tests/acceptance/testall

```diff
@@ -617,6 +617,12 @@ export CFENGINE_TEST_OVERRIDE_WORKDIR TEMP CFENGINE_TEST_OVERRIDE_EXTENSION_LIBR
             then
                 RESULT=Skip
                 RESULT_MSG="${COLOR_WARNING}Skipped (Test needs work)${COLOR_NORMAL}"
+            elif egrep "R: .*$ESCAPED_TEST FLAKEY" $OUTFILE > /dev/null
+            then
+                RESULT=FLAKEY
+                TICKET="$(egrep "R: .*$ESCAPED_TEST FLAKEY" $OUTFILE | sed -e "s,.*FLAKEY/\(.*\),\1,")"
+                RESULT_MSG="${COLOR_WARNING}Flakey fail ($TICKET)${COLOR_NORMAL}"
```

Comment:
Would be nice to not print the space and empty parentheses if there is no ticket number.

Reply:
it should result in an error, with no-ticket since a ticket is required for a flakey test.

Reply:
Actually, this is following the pattern for the others and "will never happen" because the policy test framework will make the test fail and print out no-ticket.

Granted, if you "manually" print a report of SFAIL or FLAKEY you can get this no-ticket soft fail or flakey result so could be improved for [SX]FAIL and FLAKEY.